src/bp/nlu-core/training-pipeline.ts

```diff
@@ -74,7 +72,7 @@ export type ExactMatchIndex = _.Dictionary<{ intent: string; contexts: string[]
 
 type progressCB = (p?: number) => void
 
-const debugTraining = DEBUG('nlu').sub('training')
+const debugTraining = DEBUG('nlu').sub('training') // TODO: make sure logs get wired up to web process
```

Comment:
todo done ? or is it a leftover ?

Reply:
None of the above. Its a real TODO I have had on my list for long time which is still not done.